src/main/java/com/gtnewhorizons/angelica/hudcaching/HUDCaching.java

```diff
@@ -152,18 +162,24 @@ public static void renderCachedHud(EntityRenderer renderer, GuiIngame ingame, fl
         } else {
         	GLStateManager.tryBlendFuncSeparate(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA, 1, 0);
         }
-
         if (ingame instanceof GuiIngameForge) {
         	GuiIngameForgeAccessor guiForge = ((GuiIngameForgeAccessor) ingame);
         	if (renderHelmetCaptured) {
         		guiForge.callRenderHelmet(resolution, partialTicks, hasScreen, mouseX, mouseY);
+                if (ModStatus.isHoloInventoryLoaded){
+                    Renderer.INSTANCE.angelicaOverride = false;
+                    RenderGameOverlayEvent fakeEvent = new RenderGameOverlayEvent(partialTicks, resolution, mouseX, mouseY);
```

Comment:
Does this event get mutated?  Can we make one and reuse it like we did with the biome color event?

Reply:
Nope, HoloInventory only checks the type and uses the partialTicks for rendering